app/views/collections/_show_actions.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if can? :edit, @presenter.id %>
+  <h2 class="non lower">Actions</h2>
+  <p>
+      <%= link_to "Edit", collections.edit_collection_path, class: 'btn btn-default' %> &nbsp;&nbsp;
```

Comment:
Is this going to be a broken link?  (Since the edit action in the collections controller is set to raise an exception)